File: bench/fib.js
function fib(n) { return n < 2 ? n : fib(n - 1) + fib(n - 2); }

const n      = 28;
const t0     = Date.now();
const result = fib(n);
const t1     = Date.now();

console.log(`BENCH_MS: ${t1 - t0}`);
console.log(`result=${result}`);

File: bench/fib.scala
//> using scala 3

def fib(n: Int): Int = if n < 2 then n else fib(n - 1) + fib(n - 2)

@main def run(): Unit =
  val n      = 28
  val t0     = System.nanoTime()
  val result = fib(n)
  val t1     = System.nanoTime()
  println(s"BENCH_MS: ${(t1 - t0) / 1000000}")
  println(s"result=$result")
